apps/analysis/models.py

```diff
@@ -0,0 +1,77 @@
+from django.db import models
+from django.contrib.postgres.fields import JSONField
+
+from user.models import User
+from project.models import Project
+from entry.models import Entry
+
+
+class Analysis(models.Model):
+    title = models.CharField(max_length=255)
+    team_lead = models.ForeignKey(
+        User,
+        blank=True,
+        null=True,
+        on_delete=models.CASCADE,
+    )
+    created_on = models.DateField(blank=True, null=True)
+    analysis_pillars = models.ManyToManyField(
+        'AnalysisPillar',
+        blank=True,
+    )
```

Comment:
We don't need m2m here. Use Analysis in AnalysisPillar as a foreign key with CASCADE.